api-reference/v1.0/resources/groups_overview.md

```diff
@@ -0,0 +1,125 @@
+# Working with groups in Microsoft Graph
+
+Groups are collections of [users](user.md) and other principals who share access to resources in Microsoft services or in your app. Microsoft Graph provides you with APIs to create and manage a variety of different types of groups and group functionality to suit your scenario needs. All operations in Microsoft Graph on groups require administrator consent.
+
+> **Note**: Groups can only be created through work or school accounts, and are not supported for personal Microsoft accounts.
+
+| Type              | Use case | groupType | mail-enabled | security-enabled | Creation possible through API? |
+|-------------------|----------|-----------|--------------|------------------|--------------------------------|
+| Office 365 groups | Facilitating user collaboration with shared Microsoft online resources. | ["Unified"] | true | false | yes | [user](user.md) | 
+| Security groups | Controlling user access to in-app resources. | [] | false | true | yes |
+| Mail-enabled security groups | Controlling user access to in-app resources, with a shared group mailbox. | [] | true | true | no |
+| Distribution groups | Distributing mail to the members of the group. It is recommended to use Office 365 groups due to the richer set of resources it provides. | [] | true | false | yes |
+
+## Office 365 groups in Outlook
+The power of Office 365 groups is in its collaborative nature, perfect for people who work together on a project or a team. They are created with resources that members of the group share including:
+
+- Outlook conversations
+- Outlook calendar
+- SharePoint files
+- OneNote notebook
+- SharePoint team site
+- Planner
+- Intune device management
+
+Other resources that are created but not accessible through the API include: 
+
+- Dynamics CRM
+- Microsoft Social Engagement
+- Microsoft Stream
+- Microsoft StaffHub
+
+### Example of Office 365 group in Outlook
+```http
+
+{
+    "@odata.context": "https://graph.microsoft.com/v1.0/$metadata#groups/$entity",
+    "id": "4c5ee71b-e6a5-4343-9e2c-4244bc7e0938",
+    "deletedDateTime": null,
+    "classification": "MBI",
+    "createdDateTime": "2016-08-23T14:46:56Z",
+    "description": "This is an Outlook group",
```

Comment:
Can we change the displayname, description and mailnickname to not say "Outlook group" and say something like "Office 365 group"?